content/id/docs/concepts/configuration/secret.md

```diff
@@ -559,7 +559,7 @@ apakah  terdapat perubahan pada Secret yang telah di-_mount_. Meskipun demikian,
 proses pengecekan ini dilakukan dengan menggunakan _cache_ lokal untuk mendapatkan _value_ saat ini 
 dari sebuah Secret. Tipe _cache_ yang ada dapat diatur dengan menggunakan 
 (_field_ `ConfigMapAndSecretChangeDetectionStrategy` pada
-[_struct_ KubeletConfiguration](https://github.com/kubernetes/kubernetes/blob/{{< param "docsbranch" >}}/staging/src/k8s.io/kubelet/config/v1beta1/types.go)).
+[_struct_ KubeletConfiguration](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/kubelet/config/v1beta1/types.go)).
```

Comment:
I think it is better to sync the link with EN https://github.com/kubernetes/website/blob/eae27bc59d05e4c5d91101ca286e17fd16dc2318/content/en/docs/concepts/configuration/secret.md?plain=1#L372-L373

Reply:
Agree with @habibrosyad

Reply:
Thanks for your review @habibrosyad & @ariscahyadi ! I applied your comments 🙂